auslib/test/web/test_client.py

```diff
@@ -919,22 +920,23 @@ def testEmptySnippetOn404(self):
     def testEmptySnippetOn500(self):
         with mock.patch('auslib.web.views.client.ClientRequestView.get') as m:
             m.side_effect = Exception('I break!')
-            ret = self.client.get('/update/4/b/1.0/1/p/l/a/a/a/a/1/update.xml')
-            self.assertUpdatesAreEmpty(ret)
+            with self.assertRaises(Exception) as exc:
+                self.client.get('/update/4/b/1.0/1/p/l/a/a/a/a/1/update.xml')
+                self.assertEqual('I break!', exc.exception.value())
 
     def testSentryBadDataError(self):
         with mock.patch("auslib.web.views.client.ClientRequestView.get") as m:
             m.side_effect = BadDataError("exterminate!")
-            with mock.patch("auslib.web.base.sentry") as sentry:
+            with mock.patch("auslib.web.base.sentry"), self.assertRaises(BadDataError) as exc:
                 self.client.get("/update/4/b/1.0/1/p/l/a/a/a/a/1/update.xml")
-                self.assertFalse(sentry.captureException.called)
+                self.assertEqual('exterminate!', exc.exception.value())
```

Comment:
It's important to check sentry.captureException.called, because these tests' purpose is to verify that Sentry handling is correct. Just like the previous comment, you'll need to do that outside of the context manager, though.